init-tools.cmd

```diff
@@ -0,0 +1,79 @@
+@if "%_echo%" neq "on" echo off
```

Comment:
Is that a good enough reason?

1. This file is duplicated across repos, and trying to keep it "in sync" isn't going to be possible.
2. We are already changing this file, and in some cases pretty significantly (using dotnet-install, using the CHANNEL from the branchinfo.txt file, ).

---
In reply to: [68847295](https://github.com/dotnet/cli/pull/3719#discussion_r68847295) [](ancestors = 68847295)